### PR TITLE
Preparing 0.2

### PR DESCRIPTION
**Pull request by alanz**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,121 @@
 # Changelog for haskell-language-server
 
+## 0.2
+
+- Use cabal-plan from Hackage
+([#185](https://github.com/haskell/haskell-ide-engine/pull/185) by @georgefst)
```
Comment (fendor) [inline ChangeLog.md:6]: Very sure this is the wrong url :)
```suggestion
([#185](https://github.com/haskell/haskell-language-server/pull/185) by @georgefst)
```

File: ChangeLog.md [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+([#95](https://github.com/haskell/haskell-ide-engine/pull/95) by @jneira)
+
+- Fix haddock parsing errors
+([#92](https://github.com/haskell/haskell-ide-engine/pull/92) by @jneira)
+
+- Update for haskell-lsp 0.22
+([#89](https://github.com/haskell/haskell-ide-engine/pull/89) by @alanz)
+
+- Get building with ghc-8.10
+([#83](https://github.com/haskell/haskell-ide-engine/pull/83) by @bubba)
```
Comment (fendor) [inline ChangeLog.md:117]: Everywhere else, too :)

File: ChangeLog.md (hunk 1 above)
Comment (alanz) [reply ChangeLog.md:6]: oops, copy and paste. I checked the structure, not the detail. Thanks.

**Comment by alanz** [comment]
> It would be nice if there was a section of what is really different now for end-users.
> So they know which bugs are fixed, and what got improved

I agree, but we are not going to do that for this version.
What I have seen some projects do is to include an update to the ChangeLog with the commit, so it gets updated as we go.  But only for user-visible changes.

**Pull request merged**